XLForm/XL/Descriptors/XLFormRowDescriptor.m

```diff
@@ -132,7 +143,8 @@ -(XLFormBaseCell *)cellForFormController:(XLFormViewController * __unused)formCo
                 NSString *folderName = [components firstObject];
                 NSString *bundlePath = [[[NSBundle mainBundle] resourcePath] stringByAppendingPathComponent:folderName];
                 bundle = [NSBundle bundleWithPath:bundlePath];
-            } else {
```

Comment:
This change does not follow the ObjC guidelines. For example [here](https://github.com/raywenderlich/objective-c-style-guide#spacing)

Reply:
Same happens several times below